docs/converting.asciidoc

```diff
@@ -12,41 +12,41 @@ Before you start a conversion, be sure that you understand the basics below.
 
 [float]
 [[core-or-ext]]
-=== Core and extended fields
+==== Core and extended fields
 
-* *Core fields.* Fields that are most common across all use cases are called *core fields*. 
+* *Core fields.* Fields that are most common across all use cases are called *core fields*.
 +
 These generalized fields are used by analysis content
 (searches, visualizations, dashboards, alerts, machine learning jobs, reports)
 across use cases. Analysis content designed to operate on these
-fields should work properly on data from any relevant source. 
+fields should work properly on data from any relevant source.
 +
-Focus on populating these fields first. 
+Focus on populating these fields first.
 
-* *Extended fields.* Any field that is not a core field is called an *extended field*. 
+* *Extended fields.* Any field that is not a core field is called an *extended field*.
 Extended fields may apply to more narrow use cases, or may be more open
 to interpretation depending on the use case. Extended fields are more likely to
 change over time.
 
-Each {ecs} <<ecs-fields,field>> in a table is identified as core or extended.
+Each {ecs} <<ecs-field-reference,field>> in a table is identified as core or extended.
 
 [float]
-[[ecs-comv]]
-=== An approach to mapping an existing implementation
+[[ecs-conv]]
+==== An approach to mapping an existing implementation
 
 Here's the recommended approach for converting an existing implementation to {ecs}.
 
 . Start with Core fields.
 +
 Populate core fields first. Look at your set of event fields, and find
```

Comment:
Remove "Populate core fields first."  as it is redundant to the "Start with Core fields" above.